### ISSUE TITLE
Quest Error messages [Gorgrom the Dragon-Eater (10802)]

### ISSUE THREAD

**Issue by EricMichaelLP** [Questie/Questie#2983]
## Bug description
Aquired quest [Gorgrom the Dragon-Eater (10802)] from Commander Haephus Stonewall in Evergrove - Blade's Edge Mountains, and got this error message.
Decided to check my questie and noticed it was outdated as a new version was released some hours ago. Updated and restarted game, still had same error message. 

## Screenshots
![Uten navn](https://user-images.githubusercontent.com/78097915/123570116-e1b2fe80-d7c7-11eb-948c-c39c169a161a.png)

## Questie version
Questie Minimap Icon says --- v6.3.14
Questie.toc file says --- Version: 6.4.1

**Comment by BreakBB** [comment]
Thanks for the report @EricMichaelLP 

Looks like we have either some quest data missing or a more fundamental problem 🤔 

We will also have to look into the problem, that your minimap icon does not show the correct Questie version. You restarted the game after the update, right?

**Comment by EricMichaelLP** [comment]
> Thanks for the report @EricMichaelLP
> 
> Looks like we have either some quest data missing or a more fundamental problem 🤔
> 
> We will also have to look into the problem, that your minimap icon does not show the correct Questie version. You restarted the game after the update, right?

Yeah, I did restart the game after installing the updated version. Just logged onto the game again, and still says 
"Questie v6.3.14"

I suppose the issue could come from simply importing files from the later "questie updates" and overwriting current questie files? Maybe a fresh install is what is Advised? I've simply overwritten old files in questie since I started using it (no issues) untill this quest. 

I would like to add onto the post, that I did however make note that the quest icons are displayed on the map. So is the quest objectives.
